core/components/atoms/dropdown/DropdownList.tsx

```diff
@@ -244,6 +253,8 @@ const DropdownList = (props: OptionsProps) => {
     minHeight && setMinHeight(minHeight);
   };
 
+  const _isNoResultReturned = listOptions.length === 0 && !loadingOptions && selected.length <= 0;
+
```

Comment:
Please change this variable name

Reply:
Variable name changed to **isDropdownListBlank**.